recipes/deepecho/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "deepecho" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepecho-{{ version }}.tar.gz
+  sha256: 564113deefe9b9f3b678b956342bc559711e39b1ff975c494c1bbbc8b10f0fe8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner >=2.11.1
+    - python >=3.6,<3.9
+  run:
+    - numpy >=1.18.0,<2
+    - pandas >=1.1,<1.1.5
+    - python >=3.6,<3.9
+    - pytorch >=1.4,<1.8
+    - tqdm >=4.10,<5
+
+test:
+  imports:
+    - deepecho
+    - deepecho.models
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sdv-dev/DeepEcho
+  summary: Mixed-type multivariate time series modeling with generative adversarial networks.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/rdt
```

Comment:
I want to assign our team to multiple conda-forge projects. I saw this is the suggested way here (https://conda-forge.org/docs/user/faq.html#faq-teams), but I was wondering if there is a way to change the name of our team?